### PR TITLE
[Enhancement] Print datasize column in human readable format in show tablet statement

### LINKED ISSUE #7336: [Enhancement] Print datasize column in human readable format in show tablet statement
## Enhancement
print datasize in human readable format  e.g. 1K 234M 2G
image

### PR DESCRIPTION
**Pull request by zhuxt2015**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #7336

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 2 / 8 (25.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/monitor/unit/ByteSizeValue.java|1|6|16.67%|
|:large_blue_circle:|com/starrocks/common/proc/TabletsProcDir.java|1|2|50.00%|

File: fe/fe-core/src/main/java/com/starrocks/monitor/unit/ByteSizeValue.java [hunk 1]
```diff
@@ -92,22 +92,22 @@ public double getPbFrac() {
     public String toString() {
         long bytes = getBytes();
         double value = bytes;
-        String suffix = "b";
+        String suffix = "B";
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/monitor/unit/ByteSizeValue.java:95]: If this is changed, should the units in parseBytesSizeValue and simpleParseBytesSizeValue also be changed? But then again, these two methods don't seem to work, should that be cleaned up?

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 2 / 8 (25.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/monitor/unit/ByteSizeValue.java|1|6|16.67%|
|:large_blue_circle:|com/starrocks/common/proc/TabletsProcDir.java|1|2|50.00%|

**Review by Astralidea — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Pull request merged**